cli/src/funTest/assets/semver4j-analyzer-result.yml

```diff
@@ -192,7 +192,6 @@ scanner:
       - id: "Maven:com.vdurmont:semver4j:3.1.0"
         results:
           - provenance:
-              download_time: "2020-09-30T09:27:08.894485Z"
```

Comment:
Commit message:

> This property is not needed

Please explain in the commit message *why* it is not needed, i.e. from where else the user might get the time the download was performed (roughly from the `ScannerRun`). Just the trailing "because semantically
equality does not depend on the download time" is a bit peripheral for such an important change.

An argumentation that might also help is that not all scanner implementations may actually be to provide a download time, like remote scanners that are instructed to scan a Git repository by themselves.

> Make the deserialization backwards compatible by adding

The rest of the sentence is missing.